tests/MediaTest.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+namespace RonasIT\Media\Tests;
+
+use RonasIT\Media\Models\User;
+use RonasIT\Media\Models\Media;
+use RonasIT\Media\Tests\Support\MediaTestTrait;
+use Illuminate\Http\Testing\File;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use RonasIT\Support\Traits\FilesUploadTrait;
+
+class MediaTest extends TestCase
+{
+    use FilesUploadTrait;
+    use MediaTestTrait;
+
+    protected $admin;
+    protected $user;
+    protected File $file;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->admin = User::find(1);
```

Comment:
please use static fields

```suggestion
        self::$admin ??= User::find(1);
```

Reply:
Done.